Podfile

```diff
@@ -122,6 +137,7 @@ def yosemite_pods
   stripe_terminal
   cocoa_lumberjack
   wordpress_kit
+  networking_pods
```

Comment:
In the PR description I wrote "Yosemite depends on Networking ([e.g. in Store.swift](https://github.com/woocommerce/woocommerce-ios/blob/c2bb7fef484e0cff3555ae66cbd38e8ad2e8d08b/Yosemite/Yosemite/Base/Store.swift#L3)) but the dependency was never explicit in the `Podfile`".

This change doesn't make the dependency explicit either, to be fair, but is close enough. On the top of my mind, I'm not sure whether we can make one target depend on another with our multiple projects setup. I don't think it's worth spending additional time on this right now because we may be migrating away from CocoaPods soon-ish. So, as long as the app builds, we're good.

Reply:
For the record, I think it would be possible to do it by using `abstract_target`. But I agree it's not worth spending the time.